man/man8/zpool-load.8

```diff
@@ -0,0 +1,46 @@
+.\"
+.\" CDDL HEADER START
+.\"
+.\" The contents of this file are subject to the terms of the
+.\" Common Development and Distribution License (the "License").
+.\" You may not use this file except in compliance with the License.
+.\"
+.\" You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+.\" or http://www.opensolaris.org/os/licensing.
+.\" See the License for the specific language governing permissions
+.\" and limitations under the License.
+.\"
+.\" When distributing Covered Code, include this CDDL HEADER in each
+.\" file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+.\" If applicable, add the following below this CDDL HEADER, with the
+.\" fields enclosed by brackets "[]" replaced with your own identifying
+.\" information: Portions Copyright [yyyy] [name of copyright owner]
+.\"
+.\" CDDL HEADER END
+.\"
+.\"
+.\" Copyright (c) 2019, Klara Inc. All Rights Reserved.
+.\"
+.Dd December 23, 2019
+.Dt ZPOOL-LOAD 8
+.Os Linux
+.Sh NAME
+.Nm zpool Ns Pf - Cm load
```

Comment:
This should match the subcommand name from the implementation in zpool_main.c, which has `zpool prefetch`, not `zpool load`.